### ISSUE TITLE
Show opening hours in a different format

### ISSUE THREAD

**Issue by watson** [diasdavid/awesome-hacking-spots#31]
Opening hours was recently added to the entries, but as the last column on the table. Some places have the same opening hours each week which makes it easy to add to a table, but some (like the one just added in #30) have 6 different times per week.

I'd suggest either making linebreaks for each diffrent time or moving it outside the table.

What do you guys think?

**Comment by diasdavid** [comment]
I agree to this :+1: I had [commented](https://github.com/diasdavid/awesome-hacking-spots/pull/25#issuecomment-170922149) on the PR that we might/should take that direction it if starts looking messy

**Comment by diasdavid** [comment]
Having a separate time table and a link to an always updated timetable (because they change through the year), for example to the place website, google place page, foursquare, zomato, etc, would be the better solution, otherwise we might create falsy expectations

**Comment by watson** [comment]
@diasdavid Yeah, I'm pretty worried that it's tough to keep up to date. On the other hand, some places don't advertise anything them selfs, so it's nice to have in those cases.

**Comment by watson** [comment]
I think we should also agree on a format. I suggest:

- 3 letter abbreviations for days with first letter uppercase (e.g. `Mon` instead of `Monday`)
- 24 hour clock (i.e. no am or pm)
- Colons between hours and minutes (e.g. `13:00` instead of `13.00`)

**Comment by diasdavid** [comment]
agreed, and I would prefer a 2xRow 7xColumn layer, to avoid making the scrolling incredibly large, what about:

```md
Sun | Mon | Tue | Wed | Thu | Fri | Sat
:--: | :--: | :--: | :--: | :--: | :--: | :--: | 
9:00 - 14:00 | 9:00 - 18:00 | 9:00 - 19:00 | 9:00 - 20:00 | 10:00 - 13:00 | 9:00 - 16:00 | 9:00 - 14:00 | 
````

**Comment by diasdavid** [comment]
@watson thoughts on latest proposal? Shall we proceed? :)

**Comment by watson** [comment]
Being non-american I'd personally like Monday to be the first day of the week :wink: - ISO 8601 agrees, but I'm open to suggestions.

Testing different formats:

Mon | Tue | Wed | Thu-Fri | Sat | Sun
--- | --- | --- | --- | --- | ---
9:00-14:00 | 9:00-18:00 | 9:00-19:00 | 9:00-20:00 | 10:00-13:00 | _closed_

| | |
|---|---|
| Mon | 9:00-19:00 |
| Tue | 9:00-20:00 |
| Wed | 10:00-13:00 |
| Thu | 9:00-16:00 |
| Fri-Sat | 9:00-14:00 |
| Sun | _closed_ |

- **Mon:** 9:00-19:00
- **Tue:** 9:00-20:00
- **Wed:** 10:00-13:00
- **Thu:** 9:00-16:00
- **Fri-Sat:** 9:00-14:00
- **Sun:** _closed_

```csv
Mon:      9:00-19:00
Tue:      9:00-20:00
Wed:     10:00-13:00
Thu:      9:00-16:00
Fri-Sat:  9:00-14:00
Sun:          closed
```

**Comment by diasdavid** [comment]
I like the one inside '`', takes less space. 
 
Let's go with ISO 8601

**Comment by watson** [comment]
@diasdavid the code comment version you mean?

**Comment by diasdavid** [comment]
yes :)

**Comment by watson** [comment]
@diasdavid It's a tiny bit taller though than the horizontal table, only if a lot of they days have different opening hours. I think most of the opening hours will only be a 2-3 lines in which case the code comment is gonna be smallest
